data_model/src/domain.rs

```diff
@@ -16,7 +16,6 @@ pub use self::model::*;
 use crate::{
     account::{Account, AccountsMap},
     asset::{AssetDefinition, AssetDefinitionsMap, AssetTotalQuantityMap},
-    ipfs::IpfsPath,
```

Comment:
If `IpfsPath` isn't no longer used we can remove it.
Going to create separate ticket.

Reply:
It is used by `AssetDefinition::logo`. Should `AssetDefinition::logo` also be removed?

Reply:
yes, the same logic

Reply:
Ok, will remove `AssetDefinition::logo` as part of #4417